File: people_list.cpp
#include <iostream>
#include <string>
#include <vector>
#include "person.hpp"

int main() {

    bool morepeople = true;
    std::vector<person> people;

    while (morepeople) {
        person you;
        you.getName();
        people.push_back(you);
        std::cout << "Nice to meet you, " << you.firstname << " " << you.lastname << "!\n";

        char test;
        std::cout << "Are there any more people? (y/n): ";
        std::cin >> test;

        if (test=='n') {
            morepeople = false;
        }
    }
     std::cout<<"Well, it was nice meeting you, "<<people[0].firstname <<", as well as your "<<people.size()-1<<" friends.\n";
     return 0;
}

File: people_list.py
import employee 

people = []

morepeople = True

while (morepeople):
    you = employee.employee()
    you.getName()
    print("Nice to meet you,",you.firstname,you.lastname,"\b!")
    people.append(you)
    you.getSalary()
    people.append(you)
    print(you.firstname,"'s salary is $",you.salary)
    if (input("Are there more people? (y/n) ") == 'n'):
        morepeople = False

with open("Employee_data.txt", "w") as f:
    for item in people:
        f.write(f"{item.firstname} {item.lastname}\n{item.salary}\n")

print("It was nice meeting you,",people[0].firstname,people[0].lastname,"\b, as well as your",len(people)-1,"friends.")